core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -430,7 +432,8 @@ object Boot extends IOApp with LazyLogging {
 
     for {
       blockingEc <- ExecutionContexts.fixedThreadPool[F](256) //scala.concurrent.blocking has default max extra thread number 256, so use this number to start with
-      googleStorage <- GoogleStorageService.resource[F](pathToCredentialJson, blockingEc, Some(serviceProject))
+      implicit0(it: Linebacker[F])  = Linebacker.fromExecutionContext[F](blockingEc)
```

Comment:
it's [this](https://christopherdavenport.github.io/linebacker/) library. Basically it provides this `LineBacker` typeclass which has `.blockCS` method on it, so that any blocking IO can be run on this. So instead of passing an explicit `ExecutionContext` to `GoogleStorageService`, we now just provide an implicit instance. `LineBacker` itself can be backed by any ExecutionContext, so in this case, it doesn't really change things much since we're still using fixedThreadPool

Reply:
why is making it implicit a good thing?

Reply:
That's what the doc recommends https://christopherdavenport.github.io/linebacker/. There's really no reason not to. It's an implicit new type(LineBacker), not an implicit ExecutionContext, so we wouldn't accidentally use LineBacker's ExecutionContext in places we don't intend to